src/clearAll.js

```diff
@@ -0,0 +1,8 @@
+import toDoTasks from './index.js'; // eslint-disable-line
+
+const clearAll = document.querySelector('.clearAll');
+clearAll.addEventListener('click', () => {
+  const filteredData = toDoTasks.filter((e) => e.completed === false);
+  localStorage.setItem('newTodo', JSON.stringify(filteredData));
+  window.location.reload();
+});
```

Comment:
- [ ] I have conned the repo and open in the `clearAll` branch then I run `npm i` and `npm start` but your app is not working, the add, edit, delete and delete all functions are broken, and also the local storage is not saving the information. Please 🙏🏽 fix them all for a better user experience. 👇🏽 

https://user-images.githubusercontent.com/58642949/215275840-48fb9105-848f-4d94-b970-8141373442b5.mp4

Reply:
@franclobo. Thanks for your kind review.
before submitting it i checked and it was working but what happened now I am unable to understand. can you help please.

Reply:
https://www.loom.com/share/5752fa58c87f49de9c221a49beb63f41

please check mr @franclobo